chapter_1/kmeans_1pluseps_approx.tex

```diff
@@ -0,0 +1,207 @@
+% Contributers: Noah Huber-Feely, Samuel Sharpe
```

Comment:
Overall, you did a great job in this section. However, you may want to consider breaking this section into 2 or three subsections. Maybe 1 subsection for preliminary definitions, one for the 2means result, and one for kmeans. Also try to add some intuitive explanations of the algorithms for 2means and kmeans.